examples/demo_expectations.pck

```diff
@@ -85,15 +85,15 @@ create or replace package body demo_expectations is
     ut.expect( to_char(null), 'this should fail' ).not_to( equal( to_char(null) ) );
 
     --fails on nulls not beeig equal
-    ut_assert_processor.nulls_are_equal(false);
+    ut.nulls_are_equal(false);
```

Comment:
I agree that current configuration is not good. I remove it from ut. We'll decide how to do it more approprietly later. I'm glad we've tried it, discussed and made intelligent decision.